src/node_options.cc

```diff
@@ -407,6 +407,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::allow_native_addons,
             kAllowedInEnvironment,
             true);
+  AddOption("--global-search-paths",
+            "disable global module search paths",
```

Comment:
Only the `--no-global-search-paths` disables, I suppose `--global-search-paths` keeps it enabled, right?
```suggestion
            "toggle global module search paths",
```

Reply:
This message is what shows in `node -h`:

```
  --no-deprecation            silence deprecation warnings
  --no-experimental-repl-await
                              experimental await keyword support in
                              REPL
  --no-force-async-hooks-checks
                              disable checks for async_hooks
  --no-global-search-paths    disable global module search paths
  --no-warnings               silence all process warnings
```

Using `disable` seems to align with other options. (The `no-` prefix is added by node automatically since `global-search-paths` is the default mode.)